### PR TITLE
Very minor updates to images

### PR DESCRIPTION
**Pull request by catskillmarina**
Very minor updates to images just for me to use. Added space for sslkeys as well

### CODE REVIEW

**Review by yoshinorin — CHANGES_REQUESTED** [review]
@catskillmarina

Thank you very much :)
Would you please update the badge on README? Like below.

> 9bf2e8af9765ae45eb64e2e528be59c9d96409e9

 I forgot to update it when update 4.35.0

**Comment by catskillmarina** [comment]
I'm not sure what you mean by a 'badge'. What do you need me to do.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # docker-gitbucket-orchestration
 
-![](https://img.shields.io/badge/GitBucket-v4.34.0-blue.svg)
+![](https://img.shields.io/badge/GitBucket-v4.36.0-blue.svg)
```
Comment (yoshinorin) [inline README.md]: @catskillmarina
Sorry so lack of explanation.

I put `GitBucket versions badge (label) ` to README like below.

badge

And its version is the same with the GitBucket version.
So, would you please change like below?

```diff
- ![](https://img.shields.io/badge/GitBucket-v4.36.0-blue.svg)
+ ![](https://img.shields.io/badge/GitBucket-v4.35.3-blue.svg)
```

Thank you :)

**Comment by catskillmarina** [comment]
Badge is updated as you requested.

**Review by yoshinorin — APPROVED** [review]
Thank you :)

**Pull request merged**